core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -117,7 +118,7 @@ protected NetworkParameters(Network network) {
      * network rules in a soft-forking manner, that is, blocks that don't follow the rules are accepted but not
      * mined upon and thus will be quickly re-orged out as long as the majority are enforcing the rule.
      */
-    public static final int BIP16_ENFORCE_TIME = 1333238400;
+    public static final Instant BIP16_ENFORCE_TIME = Instant.ofEpochSecond(1333238400);
```

Comment:
Since this is `public`, maybe it should be deprecated and replaced with `BIP16_ENFORCE_INSTANT`?

Reply:
I decided against it, as it's extremely unlikely someone is using that constant.

And migration is very straightforward.

Reply:
Fair enough.